webpush/push.go

```diff
@@ -80,6 +116,8 @@ func Send(client *http.Client, sub *Subscription, message, token string) (*http.
 	}
 
 	req, err := NewPushRequest(sub, message, token)
+	// Default TTL
+	req.Header.Add("ttl", "0")
```

Comment:
Missed that - it has to be removed ( as well as the todo to make it variable :-)

Since the spec requires it and doesn't have a default, I think it makes sense to have it as a request parameter.